branchpro/apps/inference_dash_app.py

```diff
@@ -124,11 +157,12 @@ def update_posterior_figure(*args):
 def calculate_posterior(*args):
```

Comment:
```suggestion
@app.app.callback(
    Output('posterior_storage', 'children'),
    [Input(s, 'value') for s in sliders],
    State('data_storage', 'children'),
)
def calculate_posterior(*args):
    """Calculate the posterior distribution.
    """
    epsilon, mean, stdev, tau, central_prob, data_json = args
```
to prevent this callback running twice when data is uploaded, which can potentially cause problems